apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.IntegrationTests/JobsTest.cs

```diff
@@ -307,6 +307,65 @@ public void DeleteJob()
             client.DeleteJob(jobToFind.Reference);
         }
 
+        [Fact]
+        public void LoadJob_CreateNewSession()
+        {
+            string sourceUri = "gs://cloud-samples-data/bigquery/us-states/us-states.parquet";
+
+            var client = BigQueryClient.Create(_fixture.ProjectId);
+            var tempTableName = "TempTestTable";
+            var tableRef = client.GetTableReference(SessionData.TempDatasetId, tempTableName);
+
+            var options = new CreateLoadJobOptions { SourceFormat = FileFormat.Parquet, CreateSession = true };
+            var job = client.CreateLoadJob(sourceUri, tableRef, schema: null, options: options);
+            job.PollUntilCompleted().ThrowOnAnyError();
+
+            var sessionId = job.Statistics.SessionInfo.SessionId;
+            Assert.NotNull(sessionId);
```

Comment:
Do we know if this *would* be null with CreateSession = true? It might be worth another test - but that can be in a separate PR.